python/tests/test_account.py

```diff
@@ -283,6 +283,15 @@ def test_get_set_profile_image_simple(self, ac1, data):
         chat.remove_profile_image()
         assert chat.get_profile_image() is None
 
+    def test_no_draft_if_cant_send(self, ac1):
+        """Tests that no quote can be set if the user can't send to this chat"""
+        device_chat = ac1.get_device_chat()
+        msg = Message.new_empty(ac1, "text")
+        device_chat.set_draft(msg)
+
+        assert not device_chat.can_send()
+        assert device_chat.get_draft() == None
```

Comment:
tests/test_account.py:293:40: E711 comparison to None should be 'if cond is None:'